recipes/android-ndk/all/test_package/conanfile.py

```diff
@@ -1,9 +1,11 @@
 import os
-from conans import ConanFile, tools, CMake
+from conan import ConanFile
```

Comment:
Might be a good idea to fully update test package to conan v2 and add a test_v1_package?

Reply:
We can add it in a second momento, because today this patch is blocking from compile some packages

Reply:
I would love a fully update, but I prefer to do it in a second PR. This PR is about fixing an issue which is blocking users. Those imports are required due the v2 linter.

Reply:
Thinking again, the change is minimal, so I decided to add test_v1_package too. Thanks for reviewing @SpaceIm